### PR TITLE
update PixelShuffle and PRelu api in vision.py and activation.py；test=document_fix

### PR DESCRIPTION
**Pull request by liyongchao911**
### PR types
 Others 

### PR changes
Docs 

### Describe
1. using paddle.randn and paddle.to_tensor to replace the numpy in some api
2. fix some format issue

paddle/docs：https://github.com/PaddlePaddle/docs/pull/5192

### CODE REVIEW

File: python/paddle/nn/functional/activation.py [hunk 1]
```diff
@@ -490,17 +490,17 @@ def prelu(x, weight, data_format="NCHW", name=None):
...
                                [-5.0,  6.0,  7.0, -8.0],
-                               [ 6.0,  7.0,  8.0,  9.0]]]], 'float32')
-            x = paddle.to_tensor(data)
-            w = paddle.to_tensor(np.array([0.25]).astype('float32'))
-            out = F.prelu(x, w)
+                               [ 6.0,  7.0,  8.0,  9.0]]]], dtype='float32')
+
+            w = paddle.to_tensor([0.25], dtype='float32')
+            out = F.prelu(data, w)
+            out
```
Comment (SigureMo) [inline python/paddle/nn/functional/activation.py]: `print(out)` 吧

**Review by SigureMo — CHANGES_REQUESTED** [review]
其余按照中文那边改一下吧～

另外中英文最好互链一下，方便跳转

**Review by Ligoml — DISMISSED** [review]
LGTM

**Comment by luotao1** [comment]
```
2022-08-26 15:25:22 1 sample codes ran failed
2022-08-26 15:25:22 samplecode_temp/paddle.nn.RReLU_example.py
2022-08-26 15:25:22 Mistakes found in sample codes. Please recheck the sample codes.
```

**Review by Ligoml — APPROVED** [review]

**Pull request merged**